src/CMakeLists.txt

```diff
@@ -848,5 +854,6 @@ endfunction(smoke_test)
 smoke_test(osh_box NONE 1 1 1 2 2 2 box.osh)
 smoke_test(osh_scale osh_box box.osh 100 box_100.osh)
 smoke_test(osh2vtk osh_scale box_100.osh box_100_vtk)
+smoke_test(osh2xgc NONE ${CMAKE_SOURCE_DIR}/meshes/plate_6elem.osh plate_6elem_xgc)
```

Comment:
Given this is a utility that will be used be only a handful of users/applications, I don't think it is a great fit for a smoke test.